preferencesfx/src/main/java/com/dlsc/preferencesfx/view/PreferencesFxPresenter.java

```diff
@@ -42,7 +43,18 @@ public void initializeViewParts() {
    */
   @Override
   public void setupEventHandlers() {
-
+    //When the Window is closed, the settings are saved beforehand.
+    preferencesFxView.sceneProperty().addListener((observable, oldScene, newScene) -> {
```

Comment:
also add a comment detailing the need for listening to the scene property (because the scene is not defined at this point)

Reply:
👍